packages/eslint-config-kibana/restricted_globals.js

```diff
@@ -0,0 +1,61 @@
+// copied from https://github.com/facebookincubator/create-react-app/blob/2e82ebb3371731a5c4e346f310848ddb23fd0976/packages/eslint-config-react-app/index.js#L24
+module.exports = [
+  'addEventListener',
+  'blur',
+  'close',
+  'closed',
+  'confirm',
+  'defaultStatus',
+  'defaultstatus',
```

Comment:
Dost my eyes deceive me? I see two of the same line here.

Reply:
Oh, one is capital `S` and one is lowercase `s`. Huh. Why this?

Reply:
Oh, i see, this comes straight from https://github.com/facebookincubator/create-react-app/blob/2e82ebb3371731a5c4e346f310848ddb23fd0976/packages/eslint-config-react-app/index.js#L24